Prox/Prox/Models/ReviewProvider.swift

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Firebase
+
+class ReviewProvider {
+
+    let url: String
+
+    let rating: Double
+    let reviews: [String]
+    let totalReviewCount: Int
```

Comment:
To be explicit, on `ReviewProvider`, I made `url` required and all other fields (`rating`, `reviews`, `totalReviewCount`) as optional.